js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/__stories__/AutoMaterializePolicyPage.stories.tsx

```diff
@@ -16,24 +16,16 @@ const path = ['test'];
 
 export const EmptyState = () => {
   return (
-    <MockedProvider
-      mocks={[Policies.NoAutomaterializeNoFreshnessPolicy(path), Evaluations.None(path, true)]}
-    >
-      <AssetAutomaterializePolicyPage assetKey={{path}} assetHasDefinedPartitions />
+    <MockedProvider mocks={[Evaluations.None(path, true)]}>
```

Comment:
Can't really tell what `true` means here.

Reply:
<img width="684" alt="Screenshot 2024-01-17 at 4 13 43 PM" src="https://github.com/dagster-io/dagster/assets/2286579/910b4bbe-a530-4ddc-b966-2125d80f39c0">
<img width="974" alt="Screenshot 2024-01-17 at 4 13 33 PM" src="https://github.com/dagster-io/dagster/assets/2286579/32716d1a-e78c-4944-9caa-b0a1f67da905">

Reply:
Sorry, my point was that when reading the function name and the args (without being able to click through to the definition), it's unclear what `true` vs. `false` actually does. Clarifying the value being passed in by using a config object for "named" vars, using an enum value, or perhaps creating a separate `NoneSingle` method, would make this clearer. I know it's just a fixture method though, so not a huge deal. :)